web/libs/editor/src/components/Comments/Comment/CommentForm.scss

```diff
@@ -56,6 +54,8 @@
     background-color: transparent;
     color: var(--secondary-action-color);
     border-radius: var(--primary-action-border-radius);
+    padding-left: 8px;
+    padding-right: 8px;
```

Comment:
```suggestion
    padding-inline: 8px;
```

Reply:
wow! TIL, thank you! :)
I knew there should be something for this! and it's even more semantic